Improve Readability (#22688)
mdn/content: files/en-us/web/api/file_api/using_files_from_web_applications/index.md

}, false);
```

> You can style the button for opening the new file picker as you wish.

Reviewer: I think "new button" is intentional here isn't it? We just added this button element in this example, so it's "new". I don't think "new file picker" makes any more sense. Maybe:

```suggestion
You can style the {{HTMLElement("button")}} however you wish.
```

?
